server/auth/types/claims.go

```diff
@@ -14,6 +14,7 @@ type Claims struct {
 	Email              string                 `json:"email,omitempty"`
 	EmailVerified      bool                   `json:"email_verified,omitempty"`
 	ServiceAccountName string                 `json:"service_account_name,omitempty"`
+	Username           string                 `json:"preferred_username,omitempty"`
```

Comment:
Why `preferred_username` instead of `username`?

Reply:
Since `preferred_username` is a standard claim  that's why used it.

ref https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims 


do you think we should give an option to change/override it just like we do for groups i.e `customGroupClaimName`  ?

Reply:
maybe change field name to match the JSON?

Reply:
Yes we can do that or add to the docs that `perferred_username` claim will be used for username ?

What would you suggest ?

Reply:
It think qualification is good

Reply:
sorry, but I am not able comprehend what you are suggesting.

Reply:
Hey @alexec  please help on how should I  proceed here.

Reply:
I think field names should match JSON name, i.e. this should be

    PreferredUsername           string                 `json:"preferred_username,omitempty"`